### ISSUE TITLE
Youtube error from headphones

### ISSUE THREAD

**Issue by schneems** [ploopyco/headphones#38]
This is going to sound weird, but the headphones managed to make youtube inoperable for me. I kept getting this message 

![image (5)](https://github.com/ploopyco/headphones/assets/59744/10d641d6-7b46-4586-aa85-2ba4903c821a)

At first I only caught the "please restart your computer," to which I was a little offended. Since when has a problem with chrome required a computer restart? It showed up on every youtube page. Even after I shut the browser down and restarted it.

On second look, I realized it's saying I had a problem with the audio. I unplugged my ploopy headphones, refreshed the page and sure enough...it worked.

I'm not sure how to reproduce the issue. I've played plenty of youtube videos with the headphones. Maybe there's a time/heat issue or something? I recently switched to a shorter USB c cable (3-feet) and it's plugged into a thunderbolt dock to power the ploopy. The cable is laptop charging rated. I'll keep an eye out if I think other factors might affect it.

**Comment by shadow-absorber** [comment]
yeah can confirm this.... if you let the ploopy headphones not output audio for a while youtube starts failing to playback sound and so does twitch or other apps on linux too...
